terminologi-arbeidsfil.md

```
@@ -0,0 +1,96 @@
+| tjenesteeier | altinn-studio-docs | process/tasks/_index.nb.md:114 | — |
+| instans | altinn-studio-docs | (gjennomgående, 40+ forekomster) | — |
+| eksemplar | altinn-studio-docs | configuration/messagebox/create-copy/_index.nb.md:12 (én gang, i parentes) | — |
+| signeringsoppgave | altinn-studio-docs | process/tasks/_index.nb.md:78, 80, 84; process/tasks/signing/_index.nb.md:5, 17, 21, 22, 64 | — |
+| signaturoppgave | altinn-studio-docs | process/tasks/_index.nb.md:110 (2 forekomster i samme avsnitt) | — |
+| signeringssteget | altinn-studio-docs | process/tasks/signing/_index.nb.md:144, 146 | — |
+| signeringstrinn | altinn-studio-docs | process/tasks/signing/_index.nb.md:246, 248 | — |
+| signeringstrinnene | altinn-studio-docs | process/tasks/signing/_index.nb.md:248 | — |
+| bekreftelsesoppgave | altinn-studio-docs | process/tasks/_index.nb.md:45, 47, 49; process/tasks/signing/_index.nb.md:17, 46 | — |
+| bekreftelsessteg | altinn-studio-docs | configuration/process/customize/_index.nb.md:65, 69, 71 | — |
+| datasteg | altinn-studio-docs | configuration/process/customize/_index.nb.md:71 | — |
+| dataoppgave | altinn-studio-docs | process/tasks/_index.nb.md:15; _glossary.nb.md:92 | — |
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)